evap/evaluation/templates/base.html

```diff
@@ -3,171 +3,91 @@
 {% load cache %}
 {% load compress %}
 {% load evaluation_filters %}
-{% load bootstrap3 %}
-{% load staff_templatetags %}
-{% load grades_templatetags %}
-{% load results_templatetags %}
+{% load navbar_templatetags %}
 
 {% endspaceless %}<!DOCTYPE html>
 <html>
 <head>
     {% block header %}
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <title>{% block title %}EvaP{% endblock %}</title>
 
         {% compress css %}
-            <link rel="stylesheet" href="{% get_static_prefix %}css/jquery.ui.css" />
-            <link type="text/less" rel="stylesheet" href="{% get_static_prefix %}css/bootstrap.less" />
+            <link type="text/x-scss" rel="stylesheet" href="{% get_static_prefix %}font-awesome/scss/font-awesome.scss" />
```

Comment:
i would either, if you keep using the scss file, remove the compiled css files from the repository, or simply use the min.css and remove everything else. i think i would actually prefer the latter, we do that with all the other libs as well.

Reply:
using min.css